packages/react-router/modules/generatePath.js

```diff
@@ -10,7 +10,7 @@ const compileGenerator = pattern => {
 
   if (cache[pattern]) return cache[pattern];
 
-  const compiledGenerator = pathToRegexp.compile(pattern);
+  const compiledGenerator = pathToRegexp.compile(pattern, { pretty: true });
```

Comment:
`pretty` should be passed when calling the compiled function, not when creating it